packaging/bee.yaml

```diff
@@ -60,8 +60,8 @@ password-file: /var/lib/bee/password
 # standalone: false
 ## enable swap (default true)
 # swap-enable: true
-## swap ethereum blockchain endpoint (default "http://localhost:8545")
-swap-endpoint: https://rpc.slock.it/goerli
+## swap ethereum blockchain endpoint (default "ws://localhost:8546")
```

Comment:
Why 8546 and not 8545?

Reply:
From https://geth.ethereum.org/docs/rpc/server:
HTTP Server: The default listening port is 8545.
WebSocket Server: The default WebSocket port is 8546.